templates/hooks/db_cluster_parameter_group/sdk_read_many_post_set_output.go.tpl

```diff
@@ -1,8 +1,17 @@
-	if ko.Status.ACKResourceMetadata != nil && ko.Status.ACKResourceMetadata.ARN != nil {
+    if ko.Status.ACKResourceMetadata != nil && ko.Status.ACKResourceMetadata.ARN != nil {
         resourceARN := (*string)(ko.Status.ACKResourceMetadata.ARN)
         tags, err := rm.getTags(ctx, *resourceARN)
         if err != nil {
             return nil, err
         }
         ko.Spec.Tags = tags
-	}
+    }
+    if ko.Spec.Name != nil {
+        groupName := ko.Spec.Name
+        params, paramStatuses, err := rm.getParameters(ctx, groupName)
+        if err != nil {
+            return nil, err
+        }
+        ko.Spec.ParameterOverrides = params
+        ko.Status.ParameterOverrideStatuses = paramStatuses
+    }
```

Comment:
Not necessary to fix in this PR, but just a quick tip... you should replace four spaces with `\t` characters in all the templates. Likely your IDE or text editor just isn't using TABs when the code file isn't a `.go` file...